Nfield.Quota/QuotaFrameLevel.cs

```diff
@@ -23,11 +23,14 @@ public QuotaFrameLevel(IEnumerable<QuotaFrameVariable> variables)
         public string Name { get; set; }
 
         public ICollection<QuotaFrameVariable> Variables { get; }
+        IEnumerable<QuotaFrameVariable> IQuotaCell.Variables => Variables;
```

Comment:
There are two `Variables` properties now?

Reply:
The one on the interface is `IEnumerable` while the one on the class is `ICollection`. Maybe it's pedantic but the interface `IQuotaCell` is meant to have an immutable API as it's only used for validation.. `ICollection` is of course not immutable as it has `Add`, `Clear`, etc.

If you want I can change the interface (it's internal only), but I thought this made the intent more clear. I can see how it's confusing tho.

Reply:
no worries, just seemed odd and wanted to ask about it

Reply:
By the way, the interface is odd in general :smile: but it exists because it turns out a lot of validation rules just want to see the frame as a level (root level). That is how it's stored in the database but not how this package represents it. The interface makes it so you can write a generic validation rule that works for levels + the root level.